qt/applications/workbench/workbench/projectrecovery/recoverygui/projectrecoverywidgetview.py

```diff
@@ -55,6 +58,10 @@ def _add_data_to_table(self):
         self.ui.tableWidget.setItem(0, 0, QTableWidgetItem(row[0]))
         self.ui.tableWidget.setItem(0, 1, QTableWidgetItem(row[1]))
 
+    def closeEvent(self, _):
+        if self.editor is not None:
+            self.editor.disconnect_editor_from_sig_process()
```

Comment:
Similarly `self.editor.disconnect_from_progress_reports` for symmetry?

Reply:
Done :+1: